transform/mattermost-analytics/models/marts/sales/dim_customers.sql

```diff
@@ -0,0 +1,8 @@
+select
```

Comment:
Let's add this model to the docs.

Reply:
One minor thing I'm thinking is whether `dim_customers` is a bit too generic. If the intention is to expand this table with other sources of customers,. then it should be ok. However, if it's only meant to be for self-serve customers, perhaps we can rename it to `dim_self_serve_customers`.

Reply:
Updated docs, changed name to dim_self_serve_customers